misc/python/materialize/mzcompose.py

```diff
@@ -705,6 +730,33 @@ def run(self, workflow: Workflow) -> None:
             raise errors.Failed(f"ERROR: services didn't restart cleanly: {services}")
 
 
+@Steps.register("remove-volumes")
+class RemoveVolumesStep(WorkflowStep):
+    """
+    Params:
+      volumes: List of volume names
+    """
+
+    def __init__(self, *, volumes: List[str]) -> None:
+        self._volumes = volumes
+        if not isinstance(self._volumes, list):
+            raise errors.BadSpec(f"volumes should be a list, got: {self._volumes}")
+
+    def run(self, workflow: Workflow) -> None:
+        volumes = [
+            v["Name"] for v in workflow.composition.inspect_volumes(self._volumes)
+        ]
+        if len(volumes) != len(self._volumes):
+            raise errors.Failed(
+                f"Unable to find correct number of volumes, found: {volumes}"
+            )
+        try:
+            spawn.runv(["docker", "volume", "rm", *volumes])
+        except subprocess.CalledProcessError:
+            msg = ", ".join(self._volumes)
+            raise errors.Failed(f"ERROR: unable to remove volumes: {msg}")
```

Comment:
Volumes in Docker Compose are always `<PROJECT NAME>_<VOLUME NAME>`, AFAICT, so I think this step can simplify to:

```py
volumes = (f"{composition.name}_{v}" for v in self._volumes)
spawn.runv(["docker", "volume", "rm", *values)
```